ISSUE_TEMPLATE.md

```diff
@@ -3,10 +3,10 @@ ATTENTION! If you would like to submit an EIP and it has already been written as
 
 If you are considering a proposal but would like to get some feedback on the idea before submitting a draft, then continue opening an Issue as a thread for discussion.  Note that the more clearly and completely you state your idea the higher the quality of the feedback you are likely to receive.
 
-Keep in mind the following guidelines from [EIP-1](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md):
+Keep in mind the following guidelines from [EIP-1](https://eips.ethereum.org/EIPS/eip-1):
```

Comment:
I'd actually propose the best approach to be `./eip-1.md`. This can be picked up properly when browsing through github. The `eips.ethereum.org` renderer could also pick it up and replace it with `https://eips.ethereum.org/EIPS/eip-1`.

What do you think?

Reply:
That means that everybody currently linking to the GitHub URLs will continue to see the pages on GitHub. And when they click on the link then they will stay on GitHub.

I was thinking that eips.ethereum.org will be more friendly to look at and hoping to get more traffic there instead of GitHub. I cringe when I see academic references to a GitHub URL. I even saw one that cited the author of a prominent EIP as "GitHub, Inc" 😢 .

Reply:
I personally prefer to think about the EIP repository as being accessed through some content addressable system to ensure its integrity. Git itself can ensure that, albeit many people do not specifically link to an EIP through its commithash.

I do not consider Github nor eips.ethereum.org satisfying that requirement and keeping one or the other leans towards preferring them.

I actually hope we could move eips.ethereum.org to be a Swarm or IPFS stored and referred through an ENS name.

Reply:
In the future all the robots will kill the humans and the robots will eat each other to stay alive.

But in our lifetimes, academic citations will use URLs and DOIs to reference content.

Reply:
Are links within EIP markdowns following the academic citation format?

Reply:
Yes, I updated to https://eips.ethereum.org/EIPS/eip-1 for a few of them.

But if these few are merged then I'll make another PR that does all of them.